src/test/java/woowacrew/security/provider/TestLoginProvider.java

```diff
@@ -27,11 +27,16 @@ public Authentication authenticate(Authentication authentication) throws Authent
         Degree degree = degreeRepository.findByNumber(0)
```

Comment:
```suggestion
        Degree degree = degreeRepository.findByNumber(Degree.NONE_DEGREE)
```
0을 기존에 의미있는 상수로 바꿔주면 좋을 거 같아요 :)